test/e2e/utils/package_plugin.go

```diff
@@ -26,3 +30,15 @@ func PackageInstalledList(namespace string) (string, error) {
 func PackageInstalledListAllNamespaces() (string, error) {
 	return Tanzu(nil, "package", "installed", "list", "-A")
 }
+
+func GetBootstrapClusterDiagnostics() (string, error) {
+	return Tanzu(nil, "diagnostics", "collect", "--output-dir", "/tmp/")
+}
+
+func GetManagementClusterDiagnostics(clusterName string) (string, error) {
```

Comment:
Yes GetManagementClusterDiagnostics function is for getting management cluster logs. Currently using `GetBootstrapClusterDiagnostics` if any issues while cluster creation of both standalone and managed cluster. The command gets all the cluster logs since we are not mentioning any skip cluster option.

Reply:
Just `tanzu diagnostics collect --output-dir /tmp/` cannot collect workload cluster or management cluster data, as there's a check for the management cluster and workload cluster name in the code -

https://github.com/vmware-tanzu/community-edition/blob/b2bdd14e4deded424575d068827f604ac690d099/cli/cmd/plugin/diagnostics/pkg/collect_cmd.go#L147-L149

https://github.com/vmware-tanzu/community-edition/blob/b2bdd14e4deded424575d068827f604ac690d099/cli/cmd/plugin/diagnostics/pkg/collect_cmd.go#L185-L187

Reply:
So with just `GetBootstrapClusterDiagnostics` we can only collect the logs of pods in the kind bootstrap cluster and the logs of the kind node etc. This can help us in finding issues in the creation of standalone cluster and management cluster too as the pods in bootstrap cluster will be CAPI, CAP* pods. But if there's issue in the created cluster, we wont' get that data